src/Evolution/Systems/GeneralizedHarmonic/Equations.hpp

```diff
@@ -52,7 +52,51 @@ namespace GeneralizedHarmonic {
  * \brief Compute the RHS of the Generalized Harmonic formulation of
  * Einstein's equations.
  *
- * \details For the full form of the equations see \cite Lindblom2005qh.
+ * The evolved variables are the spacetime metric \f$g_{ab}\f$, its spatial
+ * derivative \f$\Phi_{iab}=\partial_i g_{ab}\f$, and conjugate momentum
+ * \f$\Pi_{ab}=n^c\partial_c g_{ab}\f$, where \f$n^a\f$ is the spacetime
+ * unit normal vector. The evolution equations are \cite Lindblom2005qh
```

Comment:
Thanks!! Would you mind please adding the equation number to the citation? Please squash this small change

Reply:
Okay, added. Please let me know if you're okay with the way I did it :)